doc/lsst.meas.deblender/tasks/lsst.meas.deblender.SourceDeblendTask.rst

```diff
@@ -0,0 +1,66 @@
+.. lsst-task-topic:: lsst.meas.deblender.SourceDeblendTask
+
+#################
+SourceDeblendTask
+#################
+
+``SourceDeblendTask`` Splits blended sources into individual sources.
+
+.. _lsst.meas.deblender.SourceDeblendTask-summary:
+
+Processing summary
+==================
+
+.. If the task does not break work down into multiple steps, don't use a list.
+.. Instead, summarize the computation itself in a paragraph or two.
+
+``SourceDeblendTask`` runs this sequence of operations:
+
+#. Create the task, adding necessary fields to the given schema.
+
+#. Get the psf from the provided exposure and then run deblend().
```

Comment:
“PSF” (when you are referring to a point spread function in general), or `Psf` (when you are referring to our `Psf` class specifically).